krun.py

```diff
@@ -277,9 +307,27 @@ def main():
     config = util.read_config(config_file)
     out_file = util.output_name(config_file)
 
+    mail_recipients = config.get("MAIL_TO", [])
```

Comment:
I think MAIL_TO should be optional, so if it doesn't have a value, don't send mails. [Though I suppose an empty list is equivalent...]